__tests__/app.test.js

```diff
@@ -191,6 +183,15 @@ describe("Articles Endpoint", () => {
           });
         });
     });
+    test("GET:200 - Responds with an empty array when given a valid article id with no comments on article", () => {
+      return request(app)
+        .get("/api/articles/7/comments")
+        .expect(200)
+        .then(({ body }) => {
```

Comment:
If you wanted to you can destructure the comments in one line using nested destructuring. 
it would look like this 
```
.then(({ body: { comments }  }) => {
```